release-notes.txt

```diff
@@ -0,0 +1,97 @@
+BISDN Linux
+v4.1.0 Release Notes
+Date: 2020-09-20
+
+! New in this release
+- The port LEDs on platform Celestica Questone 2A are now functional.
+
+! Changed behaviour
+- Due to FRR 8.0.1 now enforcing link IPv6 local addresses for various
+  IPv6 based protocols, IPv6 Link local addresses are now enabled by
+  default on all interfaces.
+- SSH over IPv6 will now also be blocked by default on non management
+  ports.
```

Comment:
This phrasing does not make it clear what "also" refers to. SSH? IPv6? Non-managment ports? Besides, this sounds like users need to configure extra table entries if they want to forward ssh traffic.

Reply:
How about

> SSH connections to the switch on non management ports will now also be blocked for IPv6.

Reply:
Perfect.